src/web/app/pages-student/student-home-page/student-home-page.component.spec.ts

```diff
@@ -36,4 +36,60 @@ describe('StudentHomePageComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with null fields', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with blank fields', () => {
```

Comment:
Per the previous comment, `blank` fields is actually a duplicate here. You can either remove this test case or morph it into something else, e.g. a course with no feedback session.

Reply:
Gotcha 👍